client/termsetting/termsetting.ts

```diff
@@ -877,6 +877,47 @@ export async function fillTermWrapper(tw: TermWrapper, vocabApi: VocabApi, defau
 	return tw as TermWrapper
 }
 
+export async function fillTermWrappers(
+	tws: TermWrapper[],
+	vocabApi: VocabApi,
+	defaultQByTsHandler?: DefaultQByTsHandler
+) {
+	{
+		const ids: Array<string | undefined> = []
+		for (const tw of tws) ids.push(tw.id)
+		const terms = await vocabApi.getterms(ids)
+
+		for (const tw of tws) {
+			tw.term = terms[tw.id || tw.term.id]
```

Comment:
@airenzp 
- it's easy to use the wrong function when the only difference is the `s` at the end of one of the function names. Could it be renamed `fillTwLst()`?
- why is the single `fillTermWrapper()` not being called inside here, could this code duplication be avoided?

Reply:
i agree. is the function returning list of term wrapper objects or term json objects? they are different. if latter, should name it `getTerms()` so it's apparently different than existing `getterm()`

Reply:
The functions receives different parameters so I thought it was ok to name it similar, so it is clear what it does. When making fillTermWrappers I tried to minimize changes to avoid breaking changes. It was not so straightforward as calling fillTermWrapper inside a loop.

Reply:
@xzhou82 the function returns a list of term wrappers

Reply:
@airenzp 
- A one letter difference in long function names is confusing, makes IDE auto-completion and find-replace prone to error. Please change.

- having two server routes that do almost the same thing is not ideal, it increases maintenance/testing/support.  Maybe the `fillTermWrapper()` should call the new function and supply it with an array of its own argument? It would be bad if the "filler" logic and expectations start to diverge between these two functions, having to duplicate logic in two place (like create isAtomic) is not good. Maybe you can extract what's common between them?

Reply:
I think I have addressed now all the issue @siosonel. Please check.

Reply:
About unifying endpoints, getterm is used in many places. I am looking at it. This is a very central function, so I am cautious about introducing changes.

Reply:
@siosonel now I have unified the endpoints. Also all the unit and integration tests pass. Please check.